packages/driver/cypress/e2e/e2e/csp-headers.cy.js

```diff
@@ -0,0 +1,62 @@
+describe('csp-headers', () => {
+  // Currently unable to test spec based config values for stripCspDirectives
+  if (cy.config('stripCspDirectives') === 'all') {
```

Comment:
Since we cannot set the `stripCspDirectives` on a per spec/test basis due to https://github.com/cypress-io/cypress/issues/21151, I've added both tests here for stripping the CSP header, and not stripping, and running them through an `if` statement. In the future, we can run both these tests and set the config value for each test.